src/Grpc.AspNetCore.Server/Internal/PipeExtensions.cs

```diff
@@ -171,7 +171,7 @@ private static bool TryReadHeader(in ReadOnlySequence<byte> buffer, out bool com
             }
             else
             {
-                Span<byte> headerData = stackalloc byte[HeaderSize];
+                Span<byte> headerData = new byte[HeaderSize];
```

Comment:
Yikes https://github.com/dotnet/roslyn/issues/35764. A broken Roslyn shouldn't make its way into our SDK build though, need to figure out why we are getting a bad compiler.

Reply:
The PR for the fix is already approved. Could just wait for the fix before this change goes in. and revert this workaround.

Reply:
This isn't blocking anything. Lets wait until the SDK is fixed.